manifests/istio-telemetry/kiali/templates/kiali-kialicr.yaml

```diff
@@ -0,0 +1,118 @@
+{{- if .Values.kiali.enabled }}
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: {{ .Release.Namespace }}
```

Comment:
I'd like to decouple the release namespace (`istio-system`) from where this cr is installed. Is that viable? For example, a default of `istio-extensions` makes sense to me. Another option is `istio-integrations`.

Reply:
Yes. In fact that is how it works when you install Kiali/Kiali Operator via its own installer (both via OLM or the installer from  `https://git.io/getLatestKialiOperator` - see https://kiali.io/documentation/getting-started/). But for simplicity, it was implemented this way. See the description of this PR for an alternative. Specifically where it says:
```
NOTE: to support multi-tenancy, there really should only be a
single operator installed in the entire cluster. That one operator
should be installed in a separate namespace within the cluster
(which istioctl should create) and that one operator's Deployment
env var WATCH_NAMESPACE should be set to "". This allows a
single cluster-wide operator to install multiple Kiali instances
anywhere in the cluster. This PR does NOT implement things this
way, but it could be changed to do that. But since this feature is
considered for demo purposes only, to keep it simple I
did not implement it that way.
```